cmd/kueuectl/app/list/list_workload_test.go

```diff
@@ -381,14 +403,20 @@ wl1               j1         lq1          cq1            ADMITTED
 					Creation(testStartTime.Add(-2 * time.Hour).Truncate(time.Second)).
 					Conditions([]metav1.Condition{
 						{
-							Type:   kueue.WorkloadFinished,
-							Status: metav1.ConditionFalse,
+							Type:               kueue.WorkloadAdmitted,
+							Status:             metav1.ConditionTrue,
+							LastTransitionTime: metav1.NewTime(testStartTime.Add(-1 * time.Hour).Truncate(time.Second)),
+						},
+						{
+							Type:               kueue.WorkloadFinished,
+							Status:             metav1.ConditionFalse,
+							LastTransitionTime: metav1.NewTime(testStartTime.Truncate(time.Second)),
 						},
 					}...).
 					Obj(),
 			},
-			wantOut: `NAME   JOB TYPE   JOB NAME   LOCALQUEUE   CLUSTERQUEUE   STATUS     POSITION IN QUEUE   AGE
-wl1               j1         lq1          cq1            FINISHED                       60m
+			wantOut: `NAME   JOB TYPE   JOB NAME   LOCALQUEUE   CLUSTERQUEUE   STATUS     POSITION IN QUEUE   DURATION   AGE
+wl1               j1         lq1          cq1            FINISHED                       60m        60m
```

Comment:
Add a test cases / or modify the exiting ones to cover the two scenarios:
1. the workload is re-admitted
2. the workload finished in the past (last transition time < now)

Reply:
> the workload finished in the past (last transition time < now)

We have a few tests with this condition. For example:

https://github.com/kubernetes-sigs/kueue/blob/94319f97aaf042e803552f21b00da475d22f6315/cmd/kueuectl/app/list/list_workload_test.go#L223-L227

Reply:
Awesome, i missed that

Reply:
> the workload is re-admitted

I'm not clear on the difference between `admitted` and `re-admitted`, especially in the context of conditions. Are you asking if we should test with Admitted=false?

Reply:
Yeah, I think we don't need to test re-admission actually. My intention was to just test when lastTransitionTime for Admitted > creation time. Do we have that covered already too?

Reply:
> Are you asking if we should test with Admitted=false?

This test cases I missed. Thanks!

Reply:
> My intention was to just test when lastTransitionTime for Admitted > creation time.

I thought that it can't be possible... But I think we can add this case. 
What should we need to print on this way? WDYT? Just empty line?

Reply:
I just want to make sure the very basic scenario is covered, when Admitted=True, and the lastTransitionTime is later than creation time of the workload. This would be in practice almost all the time, but since we mock the timer, I'm not clear if this is tested.